### PR TITLE
Separated tubular motor module ZB-Sm with extended functional

### PR DESCRIPTION
**Pull request by kirovilya**
image
image

### CODE REVIEW

File: converters/fromZigbee.js [hunk 1]
```diff
@@ -8575,6 +8575,71 @@ const converters = {
             return result;
         },
     },
+    zb_sm_tuya_cover: {
```
Comment (Koenkk) [inline converters/fromZigbee.js:8578]: Could you move both the added toZigbee and fromZigbee converter into `tuya.js` (`fzLocal`/`tzLocal`) We have problems of these files getting too big.

Comment (kirovilya) [reply converters/fromZigbee.js:8578]: ok

**Comment by Koenkk** [comment]
thanks!

File: devices/tuya.js [hunk 2]
```diff
@@ -2426,4 +2557,28 @@ module.exports = [
...
+        model: 'ZB-Sm',
+        vendor: 'TuYa',
+        description: 'Tubular motor',
+        fromZigbee: [fzLocal.zb_sm_cover, fz.ignore_basic_report],
+        toZigbee: [tzLocal.zb_sm_cover],
+        onEvent: tuya.onEventSetTime,
+        exposes: [
+            e.cover_position().setAccess('position', ea.STATE_SET),
+            exposes.enum('goto_positon', ea.SET, ['25', '50', '75', 'FAVORITE']),
+            exposes.enum('running_state', ea.STATE, ['OPENING', 'CLOSING', 'STOPPED']),
```
[the remark this answers is not in the document]
Comment (kirovilya) [reply devices/tuya.js:2571]: I can do - haven't used it anywhere yet

**Pull request merged**